sunpy/spectra/spectrogram.py

```diff
@@ -27,6 +27,8 @@
 from sunpy.spectra.spectrum import Spectrum
 from sunpy.extern.six.moves import zip, range
 
+from details_plots import log_10_product
```

Comment:
from matplotlib import pyplot as plt

doesn't this give the code access to log_10_product?